moveit_core/collision_detection_bullet/CMakeLists.txt

```
@@ -1,29 +1,39 @@
 set(MOVEIT_LIB_NAME moveit_collision_detection_bullet)
 
 add_library(${MOVEIT_LIB_NAME}
-  src/collision_common.cpp
   src/collision_robot_bt.cpp
   src/collision_world_bt.cpp
+  src/tesseract/bullet_utils.cpp
+  src/tesseract/bullet_discrete_bvh_manager.cpp
 )
 set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
 
-target_link_libraries(${MOVEIT_LIB_NAME} moveit_collision_detection ${catkin_LIBRARIES} ${urdfdom_LIBRARIES} ${urdfdom_headers_LIBRARIES} ${LIBFCL_LIBRARIES} ${Boost_LIBRARIES})
+target_link_libraries(${MOVEIT_LIB_NAME} moveit_collision_detection
+    moveit_collision_detection_fcl ${catkin_LIBRARIES} ${urdfdom_LIBRARIES}
```

FCL shouldn't be needed here.
```suggestion
    ${catkin_LIBRARIES} ${urdfdom_LIBRARIES}
```

---

right